src/Compilers/CSharp/Test/Semantic/Semantics/TypeOfTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Missing copyright header. Alternatively, since this is just a single test, consider moving to an existing test class, say `GetSemanticInfoTests`.